CONTRIBUTING/adding_editing_information.md

```diff
@@ -59,103 +63,117 @@ git push
+- At the top of the page, you should see a pale grey box under the 'Comparing changes' text. The right-hand "head repository" drop down should show your username/fork and the branch that you worked on. Under 'base repository' you should see "ScilifelabDataCentre/pathogens-portal" and the 'base' should read 'develop'.
+- If you're happy with the list of commits and file changes shown towards the bottom of the page, click "Create pull request".
+- Write a title and description for your pull request, and then click 'Create pull request'.
+- A member of the portal team will review the pull request as quickly as possible regarding the pull request. Once approved, the changes will show on the portal.
 
-Once created, a member of the Portal team will review your changes.
-Once approved, they will be merged and published.
+## How to add a new page
 
-#### Testing with a local copy of the Portal
+The easiest way to add a new page to the portal is to [make a local copy](#using-the-web-interface). You can then navigate to the folder to which you would like to add a page (see below to gain an idea of where this might be, or check the URL of pages in the same section, as that will indicate the file structure). Create a new markdown (.md) file. We recommend copying the metadata from an existing page from the same section to initiate your page. The metadata is the piece at the top of the file that is between two lines of 3 dashes (i.e. '---'). You will have to change the metadata to reflect information about your new page, but this will ensure that your page is correctly initiated.
 
-Because the Portal is built on Hugo, it is quite easy to run a full version of the Portal on your computer and see how your changes look while doing that. See [this page for information on how to do that](https://github.com/ScilifelabDataCentre/pathogens-portal/blob/develop/CONTRIBUTING/running_a_local_copy.md).
+## Adding available data
 
-## Available datasets
+We have a database of available data/code from Sweden that can be found [in the available data section of the portal ](http://pathogens.se/datasets/all/). Multiple different types of data can be added to this database. Please note that this database is also updated approximtely monthly by the portal team.
 
-_Instructions coming soon._
+In order to add entries in the database, you should edit the `available_datasets.json` within the `data`folder. You need to update the date at the top of the file to the date that you are making the update. Then you can add your entry/entries. Please note that entries must include one author affiliated with a Swedish university, should pertain to COVID-19, and the data/code should be openly available or have clear instructions for how access can be made (in the event thatthe data could not be shared openly). Each entry should be in the format:
```

Comment:
Missing spaces between words

1. `data`folder -> `data` folder
2. thatthe data -> that the data